echo/readme.md

```
@@ -1,175 +1,175 @@
+
+### Fix for mypy extension hung up (devcontainer hang/lag)
+
+```bash
+ps -aux | grep "mypy"
+# grab all the process ids
+kill -9 <process ids seperated by spaces>
+```
+
+### (Windows Specific) Issues with the default WSL distribution that comes with Docker Desktop
+
+**Enable WSL Integration in Docker Desktop**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)